src/thirdparty/caman/instagram.js

```diff
@@ -0,0 +1,134 @@
+
+
+$(function() {
+  "use strict";
+
+  var CommandManager = brackets.getModule("command/CommandManager");
+  var Commands = brackets.getModule("command/Commands");
+  var fs = brackets.getModule("fileSystemImpl");
+  var Buffer = brackets.getModule("filesystem/impls/filer/BracketsFiler").Buffer;
+
+  function savePhoto(){
+    Caman(canvas, function () {
+      this.render(function () {
+            var image = this.toBase64();
+            var binaryDataStr = /^data:image\/png;base64,(.+)/.exec(image)[1];
+            //self.camera.savePhoto(base64ToBuffer(binaryDataStr));
+              var binary = window.atob(binaryDataStr);
```

Comment:
Not all browsers support `window.atob`, specifically I don't think IE has it.  We might already have this code somewhere in Brackets, and can expose it.  We should figure this out.